supervisor/container.go

```diff
@@ -30,9 +30,16 @@ type Container struct {
 	ownerPod *HyperPod
 }
 
-func (c *Container) run(p *Process) {
+func (c *Container) run(p *Process) error {
+	// Receives the early result of the attempt to start the container,
+	// which ensures that we do not return before EventContainerStart
+	// is emitted.
```

Comment:
we has the "container started" message in hyperstart as "ack to new container" message, but runv doesn't wait for it.